library/signals.py

```diff
@@ -37,3 +41,26 @@ def send_telegram_notification(sender, instance, created, **kwargs):
                     data={"chat_id": chat_id, "text": message},
                 )
                 print(response.json())
+
+
+def send_borrowing_notification(instance, created):
+    if created:
+        user = instance.user
+        message = (
+            f"New borrowing created:\n"
+            f"Book: {instance.book.title}\n"
+            f"Author: {instance.book.author}\n"
+            f"Due date: {instance.expected_return_date}\n"
+        )
+        if user.telegram_chat_id:
+            response = requests.post(
+                TELEGRAM_API_URL,
+                data={"chat_id": user.telegram_chat_id, "text": message},
+            )
+            print(response.json())
+
+
+@receiver(post_save, sender=Borrowing)
+def handle_new_borrowing(sender, instance, created, **kwargs):
+    if created:
+        async_task(send_borrowing_notification, instance, created)
```

Comment:
This fragment can be deleted. I moved it to the borrowing/signals.py

Reply:
Got it